critiquebrainz/frontend/external/musicbrainz_db/testing.py

```diff
@@ -0,0 +1,12 @@
+import os
+from flask_testing import TestCase
+from critiquebrainz.frontend import create_app
+
+class MBDatabaseTestCase(TestCase):
```

Comment:
Does this base class need to be based on `flask_testing`'s TestCase or a regular one from `unittest` would be enough?